### PR TITLE
[MOSIP-15671] removed config flags for crypto validations in IDA

### PR DESCRIPTION
**Pull request by manojsp12**

### CODE REVIEW

File: authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java [hunk 1]
```diff
@@ -51,17 +51,17 @@ protected String extractBioData(String dataFieldValue) throws IdAuthenticationAp
 
 	@Override
 	protected boolean isSigningRequired() {
-		return env.getProperty("mosip.ida.internal.signing-required", Boolean.class, true);
+		return false;
```
Comment (loganathan-sekaran) [inline authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java]: This should return `true`.

File: authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java [hunk 2]
```diff
@@ -51,17 +51,17 @@ protected String extractBioData(String dataFieldValue) throws IdAuthenticationAp
...
 	@Override
 	protected boolean isSignatureVerificationRequired() {
-		return env.getProperty("mosip.ida.internal.signature-verification-required", Boolean.class, true);
+		return true;
 	}
 
 	@Override
 	protected boolean isTrustValidationRequired() {
-		return env.getProperty("mosip.ida.internal.trust-validation-required", Boolean.class, true);
+		return false;
```
Comment (loganathan-sekaran) [inline authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java]: this should return 'true'

**Review by loganathan-sekaran — CHANGES_REQUESTED** [review]

File: authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java (hunk 1 above)
Comment (manojsp12) [reply authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java]: @LoganathanSekar7627 as per story "For new internal API, signature validation is enabled and all other validations are disabled."

File: authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java (hunk 2 above)
Comment (manojsp12) [reply authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java]: @LoganathanSekar7627 as per story "For new internal API, signature validation is enabled and all other validations are disabled."

Comment (manojsp12) [reply authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java]: @LoganathanSekar7627 in property file, it is disabled "mosip.ida.internal.trust-validation-required=false"

File: authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java (hunk 1 above)
Comment (loganathan-sekaran) [reply authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java]: @manojsp12 , this property is for the output signature response header,  which always should be there for all responses. This should be true.

File: authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java (hunk 2 above)
Comment (loganathan-sekaran) [reply authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java]: @manojsp12 , It seems was mistakenly disabled, it should be enabled for new internal auth. Trust validation is also relates to the signature verification which needs to be enabled for new internal auth.

Comment (manojsp12) [reply authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java]: @LoganathanSekar7627 Updated

File: authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java (hunk 1 above)
Comment (manojsp12) [reply authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java]: @LoganathanSekar7627 Updated

**Pull request merged**